unlocks.csv

```diff
@@ -372,14 +357,12 @@ Analysis|ExosuitGrapplingArmModule|0,0,ExosuitGrapplingArmModule
 Analysis|ExosuitTorpedoArmModule|0,0,ExosuitTorpedoArmModule
 Analysis|ExosuitPropulsionArmModule|0,0,ExosuitPropulsionArmModule
 Analysis|RepulsionCannon|0,0,RepulsionCannon
-Analysis|PrecursorIonEnergyBlueprint|0,1,PrecursorIonEnergyBlueprint
-Analysis|PrecursorIonEnergyBlueprint|1,1,PrecursorIonEnergyBlueprint
 Analysis|HighCapacityTank|0,0,HighCapacityTank
 Analysis|PlasteelTank|0,0,PlasteelTank
 Analysis|SwimChargeFins|0,0,SwimChargeFins
 Analysis|UltraGlideFins|0,0,UltraGlideFins
 Analysis|Compass|0,0,Compass
-Analysis|PowerUpgradeModule|0,1,PowerUpgradeModule
+Analysis|PowerUpgradeModule|0,0,PowerUpgradeModule
```

Comment:
Technically this probably could remain as standard - it's the case of picking up the Efficiency module in the Aurora drive bay, which is legitimately a normal way of getting that blueprint (the other is constructing the Cyclops)
But I think it just complicates matters, being such a niche thing in the game (we only do it in All Blueprints to avoid having to make the Cyclops)